Lib/javascript/v8/javascriptrun.swg

```diff
@@ -47,12 +49,18 @@ typedef v8::PropertyCallbackInfo<v8::Value> SwigV8PropertyCallbackInfo;
 #define SWIGV8_THROW_EXCEPTION(err) v8::ThrowException(err)
 #define SWIGV8_STRING_NEW(str) v8::String::New(str)
 #define SWIGV8_SYMBOL_NEW(sym) v8::String::NewSymbol(sym)
-#else
+#elif ((V8_MAJOR_VERSION-0) < 7 && (V8_MINOR_VERSION-0) <= 9) || ((V8_MAJOR_VERSION-0) == 7 && (V8_MINOR_VERSION-0) < 9)
```

Comment:
This check doesn't make sense to me, was it supposed to be
```suggestion
#elif ((V8_MAJOR_VERSION-0) < 7) || ((V8_MAJOR_VERSION-0) == 7 && (V8_MINOR_VERSION-0) < 9)
```
?

I must also say that all those `-0` are annoying, even if I understand that they're used in the existing code. IMHO adding a macro `SWIG_CHECK_V8_VERSION()` that could be then used as
```
#if !SWIG_CHECK_V8_VERSION(7, 9)
```
could be worth it.